pg-extend/pg_majorversion.h

```diff
@@ -0,0 +1,3 @@
+#include <pg_config.h>
+
+char pg_majorversion[2] = PG_MAJORVERSION;
```

Comment:
clang doesn't like parsing macros (that I can tell). But this makes it easy to pick out the version